### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by pkumar-singh**
**Motivation**
Fix some typos.

### CODE REVIEW

**Comment by Shoothzj** [comment]
@pkumar-singh Could you please rebase your code to paas the ci? I saw you code base is 2021 Aug

**Review by codelipenghui — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java [hunk 1]
```diff
@@ -177,7 +177,7 @@ public boolean checkOwnership(NamespaceBundle bundle) {
     }
 
     /**
-     * Method to get the current owner of the <code>ServiceUnit</code> or set the local broker as the owner if absent.
+     * Method to get the current owner of the <code>NamespaceBundle</code> or set the local broker as the owner if absent.
```
Comment (Shoothzj) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java:180]:
```suggestion
     * Method to get the current owner of the <code>NamespaceBundle</code> 
     * or set the local broker as the owner if absent.
```
line is more than 120

Comment (Shoothzj) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java:180]: @pkumar-singh could you please take a look:)

**Pull request merged**